### ISSUE TITLE
unit of the radiative loss rate

### ISSUE THREAD

**Issue by sanghyukmoon** [chianti-atomic/ChiantiPy#56]
![radloss](https://cloud.githubusercontent.com/assets/13713687/20456017/e6591430-aead-11e6-81e2-5939a315edd3.png)

The unit of the radiative loss rate in the [document](http://chiantipy.readthedocs.io/en/latest/quick_start.html#free-free-and-free-bound-continuum) looks strange to me. I think it should be something like $\Lambda/(n_e n_H)$ where \Lambda is usual volumetric cooling function.
Also, the docstring in freeBoundLoss method says it returns the cooling rate in units of ergs cm^-2 s^-1 which is again conflicting with the y-label on above plot.

**Comment by wtbarnes** [comment]
Yes I think you are correct in that the label on this plot is not correct as this gives units of erg s^-1 cm^5

The radiative loss rate is typically reported in units of erg cm^3 s^-1 such that $\Lambda n^2$ gives the volumetric radiative loss rate (for a plasma at density $n$) in units of erg cm^-3 s^-1.

I believe the free-bound loss should also have units of erg cm^3 s^-1 and so you would be correct on that count as well. @kdere can you confirm this?

**Comment by kdere** [comment]
I agree that the label is not correct.  What is plotted is Lambda and Lambda*n_e*n_H gives the radiative loss rate per unit volume erg cm^-3 s^-1.

The units for Lambda are erg cm^3 s^-1, and the same goes for the free-bound loss rate

**Comment by wtbarnes** [comment]
Ok so it seems that the units listed in the plot labels and docstrings in several places need to be fixed. @smoon24 would you mind listing all of the places where you've found these inconsistencies and adding to it as you find more? These could all be fixed in a single PR.

**Comment by sanghyukmoon** [comment]
I used Chianti for coursework by now, so I'm afraid whether if I can find more. I just have checked functions related to RadLoss and found that boundBoundLoss method in Ion.py seems to has similar error. Docstring says it returns loss rate in unit of erg cm^-3 s^-1, which should be erg cm^3 s^-1. And it might be better to add unit in the docstring of freeFreeLoss and twoPhotonLoss for consistency. Thanks!

**Comment by wtbarnes** [comment]
Thanks for your feedback @smoon24. If you happen to come across any more of these inconsistencies, feel free to list them here. We'll keep this issue open until these get fixed.

**Comment by kdere** [comment]
Yes, this is one of the things I need to get to.  I think it is also on the QuickStart page and notebook
